### ISSUE TITLE
refactor: disable source map using option 

### ISSUE THREAD

**Issue by evilebottnawi** [postcss/postcss-loader#195]
Is it possible to disable generation source map generation using options? This will speed up compilation on watch when i disable source map in webpack. It would be nice if this option was taken from the webpack config by default. If your accept i can do PR.

**Comment by michael-ciniawsky** [comment]
@evilebottnawi Yep, rigth, could you check the #177, I intend to do that for v2.0.0 since it is technically a semver major, but wainting for other related loaders I need to update first `style-loader`/ `extract-text-webpack-plugin`

**Comment by evilebottnawi** [comment]
@michael-ciniawsky can i help you with this?

**Comment by michael-ciniawsky** [comment]
@evilebottnawi Yep 😛  test if disabling sourcemaps works correctly

Sourcemap Implementation is [here](https://github.com/postcss/postcss-loader/pull/191/files#diff-168726dbe96b3ce427e7fedce31bb0bcR67)

v2.0.0 should work without `css-loader` and export a module directly (perf), but it is planned to export ES2015 Modules in the near future and `style-loader`/ `extract-text-webpack-plugin` don't support that yet. If you what to try that out you need to change this [line](https://github.com/postcss/postcss-loader/pull/191/files#diff-168726dbe96b3ce427e7fedce31bb0bcR119) to a CJS export

```diff
- const module = `export default ${JSON.stringify(css)}`
+ const module = `module.exports = ${JSON.stringify(css)}`
```

**webpack.config.js**
```js
const config = {
  module: {
    rules: [
      {
         test: /\.css$/,
         use: [ 'style-loader', 'postcss-loader' ]
      }
    ]
  }
}
```

Otherwise by using `use: [ 'style-loader', 'css-loader', 'postcss-loader' ]` you can skip that part, since `postcss-loader` will export a `{String}`  as it is the current behaviour.

**Comment by michael-ciniawsky** [comment]
`npm i -D michael-ciniawsky/postcss-loader#v2`

**Comment by michael-ciniawsky** [comment]
Fixed by #191
